app/models/institutions_archive.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require_dependency 'institution'
+
+class InstitutionsArchive < Institution
+  self.table_name = 'institutions_archives'
```

Comment:
I'm guessing this line is unnecessary

Reply:
Required as this class needs to override the value of `self.table_name` that it inherits from Institution